samples/templates/sampleSpreadsheet.php

```diff
@@ -31,7 +31,9 @@
 $helper->log('Add some data');
 $spreadsheet->setActiveSheetIndex(0);
 $spreadsheet->getActiveSheet()->setCellValue('B1', 'Invoice');
-$spreadsheet->getActiveSheet()->setCellValue('D1', Date::PHPToExcel(gmmktime(0, 0, 0, date('m'), date('d'), date('Y'))));
+$date = new DateTime('now');
```

Comment:
Should this not include the timezone ('UTC')

Reply:
It could include the timezone, but I don't think it should, i.e. I think the use of GMT in the original was a very minor design flaw. After the sample is generated, you can look at it and see the date in cell D1. It is not used for any other purpose. So, it will be a frequent occurrence that what is in cell D1 does not match the current date for the user if it uses GMT. On the other hand, if it uses local time, as the changed code does, what is in cell D1 matches the user's expectations. (Well, it matches mine if nobody else's.) It is obviously really easy to fix this if you still think it should be fixed.